### PR TITLE
add unimod accessions to mods file

### PR DESCRIPTION
**Pull request by rmillikin**

### CODE REVIEW

**Comment by rmillikin** [comment]
I've added Unimod accessions to MetaMorpheus's mods in this PR. the UniProt PTM list is missing Unimod accessions; I'll add those in another commit to this PR

File: EngineLayer/Mods/Mods.txt [hunk 1]
```diff
@@ -5,19 +5,22 @@ TG   K or N
...
 //
 ID   Phosphorylation
 TG   S or T
 PP   Anywhere.
 NL   HCD:H0 or HCD:H3 O4 P1
 MT   Common Biological
 CF   H1 O3 P1
+DR   Unimod; 21.
 //
 ID   Phosphorylation
```
Comment (trishorts) [inline EngineLayer/Mods/Mods.txt]: do you know why tyrosine is separate?

Comment (rmillikin) [reply EngineLayer/Mods/Mods.txt]: tyrosine has a diagnostic ion while S/T don't

**Comment by lonelu** [comment]
why we need the Unimod accessions?

**Comment by rmillikin** [comment]
issue #1684 . please do not review this PR yet. it's not ready for review - I need to add unit tests

**Comment by rmillikin** [comment]
should be good to go

**Pull request merged**